### PR TITLE
Fix e5s crippling blow and stepped leader timing

### PR DESCRIPTION
**Pull request by anubiann00b**
Fix timing of e5s triggers based on the notes in #1069 

I haven't validated that these are correct yet but I will tonight when I run e5s.

@Jaehyuk-Lee pinging you in since this is an addition to your work on e5s triggers (thank you!)

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e5s.js [hunk 1]
```diff
@@ -140,11 +140,11 @@
     },
     {
       id: 'E5S Crippling Blow',
-      regex: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexDe: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexFr: Regexes.ability({ id: '4BCA', source: 'Ramuh' }),
-      regexJa: Regexes.ability({ id: '4BCA', source: 'ラムウ' }),
-      regexKo: Regexes.ability({ id: '4BCA', source: '라무' }),
+      regex: Regexes.startsUsing({ id: '4BCA', source: 'Ramuh', capture: true }),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e5s.js]: can you remove capture: true? it's implicit and gets skipped everywhere else.

**Comment by quisquous** [comment]
Thanks for the fix, I've had a really busy day and wasn't able to get back to the other pull request.

File: ui/raidboss/data/05-shb/raid/e5s.js (hunk 1 above)
Comment (anubiann00b) [reply ui/raidboss/data/05-shb/raid/e5s.js]: fixed

**Comment by anubiann00b** [comment]
actually, could we wait a sec before merging--I think that timing is wrong

**Comment by quisquous** [comment]
Good now?

**Comment by anubiann00b** [comment]
yep! I'mnot 100% sure abt 3.0 seconds because prog was a blur but if it's not right it's definitely closer

**Comment by quisquous** [comment]
> yep! I'mnot 100% sure abt 3.0 seconds because prog was a blur but if it's not right it's definitely closer

I wonder if it would be better to just use the ability rather than a delay from the casting start, but I can try that in the future.

**Pull request merged**